pipelines/test-release.yml

```diff
@@ -0,0 +1,77 @@
+trigger:
+  branches:
+    exclude:
+    - '*'
+
+stages:
+- stage: Test
+  jobs:
+  - job: testextension 
+    steps:
+    - task: NLUTrain@0
```

Comment:
Use "mock" provider, rather than LUIS (so we don't incur query cost on each CI build), as done in:
https://github.com/microsoft/NLU.DevOps/commit/d64d407c3e72dd83662df7afe833a52dc265af47

Reply:
done